ergast_py/type_constructor.py

```diff
@@ -199,6 +200,18 @@ def construct_races(self, races: dict) -> list[Race]:
         """
         return [self.construct_race(race) for race in races]
 
+    def construct_time(self, time: dict) -> Time:
+        """
+        Construct a Time from a JSON dictionary
+        """
+        if time != {}:
```

Comment:
The format that's used elsewhere is to create a `_populate_missing_type` function. This will then populate any missing fields meaning that if we have empty results these will be default, and as such we don't need to perform these if checks around construction of the Time model